src/csharp/Microsoft.Spark.Worker/Processor/BroadcastVariableProcessor.cs

```diff
@@ -54,7 +54,7 @@ internal BroadcastVariables Process(Stream stream)
                     else
                     {
                         string path = SerDe.ReadString(stream);
-                        using FileStream fStream = File.Open(path, FileMode.Open, FileAccess.Read);
+                        using FileStream fStream = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.Read);
```

Comment:
Can you break this line apart and get it < 100 char?  There is a style guide [here](https://github.com/dotnet/spark/blob/master/docs/coding-guidelines/csharp-coding-style.md)

Reply:
yes, sorry about that , just pushed the formatted line.